src/redux/actions/index.js

```diff
@@ -1,12 +1,19 @@
+// import all of the constants from contants folder
 import {getData, NEW_TERMS_KEY, storeData} from '../../utils/storage';
 // // import all of the constants from contants folder
 import {
   SET_CATEGORIES,
   SET_PHRASES,
   SET_LANGUAGE_NAME,
   SET_CURRENT_CATEGORY,
+  SET_LEARNT_PHRASES,
   SET_NEW_TERMS,
 } from '../constants';
+import {
+  LEARNT_PHRASES_KEY,
+  setDataToStorage,
+  getDataFromStorage,
+} from '../../utils/storage';
```

Comment:
Looks like we have different naming function on storing data in storage (storeData and setDataToStorage), (getData, getDataFromStorage), we need to pick either of the to avoid repeating the same thing.  And update our `utils/storage`.
We could use `getDataFromStorage` , `setDataToStorage` approach - descriptive and understandable!

Reply:
And use only one `import` to import things from `utils/storage`